packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -656,15 +660,75 @@ describe('plugin-meetings', () => {
         });
       });
       describe('#startTranscription', () => {
```

Comment:
can we leave a blank line before this describe?

Reply:
Added.